test/jdk/java/foreign/TestStringEncoding.java

```diff
@@ -105,15 +105,16 @@ public void testStrings(String testString) {
 
     @Test(dataProvider = "strings")
     public void testStringsLength(String testString) {
-        Set<String> excluded = Set.of("yen", "snowman", "rainbow");
-        // This test only works for certain strings where the last character is not special
+        Set<String> excluded = Set.of("yen");
```

Comment:
I know the yen character is translated to `/` in some encodings when doing a round trip. Maybe we should just avoid this issue by switching it out for e.g. `"section \u00A7"`, which is `§` and doesn't have the same problem.